src/Tribe/Views/V2/Views/Month_View.php

```diff
@@ -398,9 +398,9 @@ static function ( $event ) use ( $date_object ) {
 			$day_data = [
 				'date'             => $day_date,
 				'is_start_of_week' => $start_of_week === $date_object->format( 'N' ),
-				'year_number'      => (int) $date_object->format( 'Y' ),
-				'month_number'     => (int) $date_object->format( 'm' ),
-				'day_number'       => (int) $date_object->format( 'd' ),
+				'year_number'      => $date_object->format( 'Y' ),
```

Comment:
Why remove the cast to `int` here?

Reply:
Because we want to make sure we keep the leading 0.

Reply:
I see, thanks for clarifying.